src/pipecat/processors/frameworks/strands_agents.py

```diff
@@ -71,9 +71,11 @@ async def process_frame(self, frame: Frame, direction: FrameDirection):
             direction: The direction of frame flow in the pipeline.
         """
         await super().process_frame(frame, direction)
-        if isinstance(frame, OpenAILLMContextFrame):
-            text = frame.context.messages[-1]["content"]
-            await self._ainvoke(str(text).strip())
+        if isinstance(frame, LLMContextFrame):
```

Comment:
I think we still need to support `OpenAILLMContextFrame`, though...we haven't deprecated it yet

Reply:
Soon! 🤞